CSS/style.css

```diff
@@ -0,0 +1,297 @@
+/* Mobile first */
+
+@media screen and (min-width: 480px) {
+
+}
+
+
+/* {
+    border: 1px solid #999;
+}*/
+
+body {
+    color: #2d2d2d;
+    font-family: "Raleway", sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+}
+
+ol, ul {
+	list-style: none;
+    -webkit-padding-start: 0;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+/* --------------- Header --------------- */
+
+header {
+    padding: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo {
+    max-width: 150px;
+    align-self: flex-start;
+    order: 1;
+}
+
+.navigation {
+    font-weight: 500;
+    align-self: center;
+    order: 2;
+}
+
+.navigation a {
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #e2574c;
+    padding: 0 1rem;
+}
+
+.navigation a:hover {
+    color: #555;
+    cursor: pointer;
+}
+
+.cart {
+    max-width: 60px;
+    align-self: flex-end;
+    order: 3;
+}
+
+/* --------------- Hero --------------- */
+
+.hero {
+    height: 550px;
+    width: 100%;
+    background:
+      linear-gradient(to bottom, rgba(0, 0, 0 ,0.3), rgba(100,100,100,0.3)),
+      url('../images/banner-girl.png') center bottom no-repeat,
+      url('../images/flower-bkgd.jpg') center top repeat-x;
+    background-size: auto, 250px, 850px;
+    background-position:
+      left 0 top 0,
+      left 50% bottom -120px,
+      left 0 top 0;
+    position: relative;
```

Comment:
Remove relative and absolute positioning on this and use flexbox properties instead.